sdk/storage/storage-file-datalake/src/Pipeline.ts

```diff
@@ -185,10 +185,11 @@ export function newPipeline(
   // The credential's policy factory must appear close to the wire so it can sign any
   // changes made by other factories (like UniqueRequestIDPolicyFactory)
 
+  const telemetryPolicy = new TelemetryPolicyFactory(pipelineOptions.userAgentOptions);
   const factories: RequestPolicyFactory[] = [
-    tracingPolicy(),
+    tracingPolicy({ userAgent: telemetryPolicy.telemetryString }),
     keepAlivePolicy(pipelineOptions.keepAliveOptions),
-    new TelemetryPolicyFactory(pipelineOptions.userAgentOptions),
+    telemetryPolicy,
```

Comment:
Just curious - is this `prettier` doing this or are you doing these refactors by hand?

Reply:
Anything with whitespace is prettier's magic. Pulling telmetryPolicy out into a constant was my doing.